lib/open_food_network/reports/variant_overrides.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module OpenFoodNetwork
+  module Reports
+    class VariantOverrides
+      def initialize(line_items:, distributor_ids:)
+        @line_items = line_items
+        @distributor_ids = distributor_ids
+      end
+
+      def indexed
+        variant_overrides.each_with_object(hash_of_hashes) do |variant_override, indexed|
+          indexed[variant_override.hub_id][variant_override.variant] = variant_override
```

Comment:
Can we break this line in two and use a good variable name to make it easier for the reader?
maybe (a bit too long but easier to understand what is going on):
```
    variant_overrides.each_with_object(hash_of_hashes) do |variant_override, indexedVariantOverridesForAllDistributors|
         indexedVariantOverridesForDistributor = indexedVariantOverridesForAllDistributors[variant_override.hub_id]
         indexedVariantOverridesForDistributor[variant_override.variant] = variant_override

```   
I think it would be good to add a comment to the original VariantOverride#indexed and/or to this method explaining what is the data structure we are creating.

Reply:
I've added a comment, but I've kept this block the way that it is. I personally find it more readable the way that it is, but it you have a strong preference for breaking it down in the way you have , let me know and I'll make the change.

Reply:
no, it's ok :+1: